src/daemon/utils_cache.c

```diff
@@ -969,11 +969,17 @@ static meta_data_t *uc_get_meta(const value_list_t *vl) /* {{{ */
     pthread_mutex_unlock(&cache_lock);                                         \
     return status;                                                             \
   }
-int uc_meta_data_exists(const value_list_t *vl,
-                        const char *key) UC_WRAP(meta_data_exists)
+int uc_meta_data_exists(const value_list_t *vl, const char *key)
+    UC_WRAP(meta_data_exists)
+
+        int uc_meta_data_delete(const value_list_t *vl, const char *key)
```

Comment:
[optional] is it intentional to indent this function `uc_meta_data_delete` is indented further right than `uc_meta_toc`? if yes, you can resolve this comment.

Reply:
Yes, that's what collectd's autoformatter does, and it's required by the upstream presubmit.

I've sent the changes daemon/utils_cache.c upstream for review, and I'd rather make it the same.